backend/src/game/cards/Card_JSON_Parser.py

```diff
@@ -0,0 +1,56 @@
+import json
+from enum import Enum
+from typing import List
+
+with open('backend\\src\\game\\cards\\KOTCards.json', "r") as json_file:
+    card_file = json.load(json_file)
+
+
+class CardType(Enum):
+    keep = 1
+    discard = 2
+    invalid = 99
+
+
+class Card(object):
```

Comment:
Please extract the Card object out as well. And make a separate file for the CardParser

Reply:
Extracted to Card.py